api/api-pastis/pastis-commons/src/main/java/fr/gouv/vitamui/pastis/common/dto/seda/SedaNode.java

```diff
@@ -57,8 +57,13 @@ public class SedaNode {
     private String definition;
     private String collection;
     private List<SedaNode> children;
+    private boolean external;
 
     public Stream<SedaNode> flattened() {
         return Stream.concat(Stream.of(this), children.stream().flatMap(SedaNode::flattened));
     }
+
+    public SedaNode getChild(String childName) {
+        return this.getChildren().stream().filter(child -> child.getName().equals(childName)).findFirst().orElse(null);
```

Comment:
ils ne sont pas susceptible d'etre null ?

Reply:
SedaNode.getChildren est toujours initialisé, au pire à une liste vide. Au pire le getChild() renvoie null, mais dans les faits ce n'est jamais le cas (car on l'appelle sur une structure connue, ou quand on itère sur des sous-noeuds, etc.)